### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Havrysh**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    order: 4;
+  }
+}
+
+@media (min-width: 900px) {
+  .block {
+    width: 33.33%;
+  }
+  .block-1 {
+    order: -1;
```
Comment (ArtemTopolskyi) [inline src/style.css:71]: don't use negative orders

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+.block-5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block-6 {
+  background-color: black;
+}
+
+@media (min-width:600px) and (max-width: 900px) {
+  .block-1 {
```
Comment (ArtemTopolskyi) [inline src/style.css:46]: same

**Review by ArtemTopolskyi — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  }
+  .block-4 {
+    order: 5;
+  }
+  .block-5 {
+    order: 4;
+  }
+  .block-6 {
+    order: 3;
+  }
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:87]: do you really need to hardcode order for every block?

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (Havrysh) [reply src/style.css]: not really)

**Review by Bohdan-Maksymiuk — APPROVED** [review]
Good job 👍

**Pull request closed without merging**